packages/js/data/src/products/index.ts

```diff
@@ -16,6 +16,8 @@ import reducer, { ProductState, State } from './reducer';
 import controls from '../controls';
 
 registerStore< State >( STORE_NAME, {
+	// @ts-expect-error There are no types for this.
+	__experimentalUseThunks: true,
```

Comment:
Nice, what benefit does this have again? ( btw not against it ).

Reply:
The Thunks API feels pretty clear to me. It allows to handle the data propagation in a simpler way.